sippy-ng/src/App.js

```diff
@@ -120,26 +120,38 @@ export default function App(props) {
   const [isLoaded, setLoaded] = React.useState(false)
   const [releases, setReleases] = React.useState([])
   const [capabilities, setCapabilities] = React.useState([])
+  const [reportDate, setReportDate] = React.useState([])
   const [fetchError, setFetchError] = React.useState('')
 
   const fetchData = () => {
     Promise.all([
       fetch(process.env.REACT_APP_API_URL + '/api/releases'),
       fetch(process.env.REACT_APP_API_URL + '/api/capabilities'),
+      fetch(process.env.REACT_APP_API_URL + '/api/report_date'),
     ])
-      .then(([releases, capabilities]) => {
+      .then(([releases, capabilities, reportDate]) => {
```

Comment:
Instead of wiring this in many places, I think it might be more appropriate to use a context and wrap the whole app like Capabilities.  What do you think?

Reply:
Agree, thanks for the pointer.  Let me review & refactor.